src/Installers/Windows/AspNetCoreModule-Setup/build/settings/common.props

```
@@ -7,6 +7,7 @@
   <!-- Define platform shortnames for use in paths without having to rely on cpp props -->
   <PropertyGroup>
     <PlatformShortname Condition="'$(PlatformShortname)' == '' AND '$(Platform)' == 'x64'">x64</PlatformShortname>
+    <PlatformShortname Condition="'$(PlatformShortname)' == '' AND '$(Platform)' == 'arm64'">arm64</PlatformShortname>
     <PlatformShortname Condition="'$(PlatformShortname)' == '' AND ('$(Platform.ToLower())' == 'win32' OR '$(Platform)' == 'x86')">x86</PlatformShortname>
```

nit, `msbuild` comparisons ignore case
```suggestion
    <PlatformShortname Condition="'$(PlatformShortname)' == '' AND ('$(Platform)' == 'win32' OR '$(Platform)' == 'x86')">x86</PlatformShortname>
```